app/views/admin/services/show.html.erb

```diff
@@ -15,6 +15,10 @@
       Visit service in app
     </a>
 
+<% if ENV['API_SUBMISSION'] == 'enabled' %>
+    <%= link_to "Show API Endpoint Submission Settings", admin_service_api_submission_index_path( :service_id => @service.service_id ), class: "govuk-button fb-govuk-button" %>
+<% end %>
+</div>
```

Comment:
```suggestion

```

I think this closing div has snuck in here. Removing it will fix the margin issue on the admin page